src/verify/serverless.yml

```diff
@@ -0,0 +1,39 @@
+service: verify
+provider:
+  name: aws
+  runtime: nodejs12.x
+  stage: "${opt:stage, 'stg'}"
+  region: "${opt:region, 'ap-southeast-1'}"
+  environment:
+    NETWORK: "${self:custom.NETWORK.${self:provider.stage}}"
+functions:
+  verify:
+    description: "Checks the validity of the certificate against the 4 checks."
+    handler: index.handler
+    events:
+      - { http: { path: /, method: post, cors: true } }
```

Comment:
why mix yml and json

Reply:
it was like this I didnt chang the syntax, I'm not sure why it was here before =)